### ISSUE TITLE
WAVE Accessibility Check: some linked images missing alt text

### ISSUE THREAD

**Issue by eff-soraya** [EFForg/starttls-frontend#167]
![screen shot 2018-06-13 at 9 58 52 am](https://user-images.githubusercontent.com/16709499/41366352-aa3734dc-6ef0-11e8-8679-b81a49b5581a.png)

I did a WAVE browser extension accessibility check: it seems that the alt-text for the social media buttons and menu logo aren't being recognized.
